CBLClient/Apps/CBLTestServer-Dotnet/TestServer/ReplicatorConfigurationMethods.cs

```diff
@@ -43,6 +43,240 @@ namespace Couchbase.Lite.Testing
+                        config.PushFilter = _default_replicator_filter_callback;
+                        break;
+                }
+            }
+            if (postBody["pull_filter"].Equals(true))
+            {
+                switch (filter_callback_func)
+                {
+                    case "boolean":
+                        config.PullFilter = _replicator_boolean_filter_callback;
+                        break;
+                    case "deleted":
+                        config.PullFilter = _replicator_deleted_filter_callback;
+                        break;
+                    case "access_revoked":
+                        config.PullFilter = _replicator_access_revoked_filter_callback;
+                        break;
+                    default:
+                        config.PullFilter = _default_replicator_filter_callback;
+                        break;
+                }
+            }
+            switch (postBody["conflict_resolver"].ToString())
+            {
+                
+                case "local_wins":
+                    config.ConflictResolver = new LocalWinsCustomConflictResolver();
+                    break;
+                case "remote_wins":
+                    config.ConflictResolver = new RemoteWinsCustomConflictResolver();
+                    break;
+                case "null":
+                    config.ConflictResolver = new NullCustomConflictResolver();
+                    break;
+                case "merge":
+                    config.ConflictResolver = new MergeCustomConflictResolver();
+                    break;
+                case "incorrect_doc_id":
+                    config.ConflictResolver = new IncorrectDocIdConflictResolver();
+                    break;
+                case "delayed_local_win":
+                    config.ConflictResolver = new DelayedLocalWinConflictResolver();
+                    break;
+                case "delete_not_win":
+                    config.ConflictResolver = new DeleteDocConflictResolver();
+                    break;
+                case "exception_thrown":
+                    config.ConflictResolver = new ExceptionThrownConflictResolver();
+                    break;
+                default:
+                    config.ConflictResolver = ConflictResolver.Default;
+                    break;
+
+            }
+            if (postBody.ContainsKey("channels"))
+            {
+                List<object> channels = (List<object>)postBody["channels"];
+                config.Channels = channels.Cast<string>().ToList();
+            }
+            if (postBody.ContainsKey("documentIDs"))
+            {
+                List<object> documentIDs = (List<object>)postBody["documentIDs"];
+                config.DocumentIDs = documentIDs.Cast<string>().ToList();
+            }
+            response.WriteBody(MemoryMap.Store(config));
+        }
+
+        public static void configurationCollection([NotNull] NameValueCollection args,
```

Comment:
Maybe rename to configureCollection?

Reply:
If i rename it here, we have to make the name change in IOS, C and Android/Java as well.